modules/web/client/src/main/scala/observe/ui/components/sequence/SequenceTable.scala

```diff
@@ -279,14 +299,30 @@ private sealed trait SequenceTableBuilder[S: Eq, D <: DynamicConfig: Eq]
       .useEffectWithDepsBy((props, _, _, _, _, _, _, _, _, _, _) => props.currentRecordedStepId):
         (_, _, _, _, _, _, _, _, _, _, odbQueryApi) => _ => odbQueryApi.refreshNighttimeVisits
       .useRef(none[HTMLTableVirtualizer])
-      .useEffectOnMountBy: (_, _, _, _, _, _, _, sequence, _, _, _, virtualizerRef) =>
-        virtualizerRef.get.flatMap: refOpt =>
-          Callback:
-            refOpt.map:
-              _.scrollToIndex(
-                sequence.indexWhere(row => CurrentHeadersRowIds.contains_(getRowId(row).value)) - 1,
-                ScrollOptions
-              )
+      .useEffectOnMountBy: (props, _, _, _, _, _, _, _, _, table, _, virtualizerRef) =>
+        val autoScrollCandidates: List[String] =
+          AlertRowId.toString +:
+            (props.runningStepId ++ props.nextStepId).map(_.toString).toList
+
+        // If sequence is not running, auto select next step.
+        Callback.when(props.runningStepId.isEmpty)(
+          props.nextStepId.map(props.setSelectedStepId).orEmpty
+        ) >>
+          scrollToRowId(virtualizerRef, table)(autoScrollCandidates)
+            .delayMs(1) // https://github.com/TanStack/virtual/issues/615
```

Comment:
`scrollToRowId` doesn't work `onMount` in the latest virtualizer version, unless `delayMs` is used.

Reply:
Interesting, even 1ms works?

Reply:
The goal is to just delay it to the next tick in the event loop. In JS it would even work with 0, but `delayMs` has an optimization where if it receives 0 it just skips the `setTimeout` and executes right away.